com.unity.multiplayer.mlapi/Runtime/Core/NetworkingManager.cs

```diff
@@ -606,6 +613,25 @@ public void StopClient()
             Shutdown();
         }
 
+        /// <summary>
+        /// Stops the running server, client or host.
+        /// </summary>
+        public void Stop()
```

Comment:
I'm not sure whether this is really a nice way to solve this. But I have run into this issue where I just wanted to have MLAPI stop the simulation and I didn't care whether I'm a server, client or host so I added this for convenience.